papaparse.js

```diff
@@ -1731,7 +1774,7 @@ License: MIT
 			}
 
 			/** Gets the delimiter character, which is not inside the quoted field */
-			function getNextUnqotedDelimiter(nextDelim, quoteSearch, newLine) {
+			function getNextUnquotedDelimiter(nextDelim, quoteSearch, newLine) {
```

Comment:
Mmm, I will be happy to merge this as a separate PR as it does not seem releated.